docs/source/migration.md

```diff
@@ -1,28 +1,50 @@
 
 ## Bumped dependencies
 
-Apollo Server 3 only supports Node 12 or newer. (Apollo Server 2 supported Node v6.) This includes [all LTS and Current versions at the time of release](https://nodejs.org/en/about/releases/). If you are using an older version of Node, you should upgrade your Node runtime before upgrading to Apollo Server 3.
+### Node.js 
+
+**Apollo Server 3 supports Node.js 12 and later.** (Apollo Server 2 supports back to Node.js 6.) This includes [all LTS and Current versions at the time of release](https://nodejs.org/about/releases/).
+
+If you're using an older version of Node, you should upgrade your Node runtime before upgrading to Apollo Server 3.
+
+### `graphql`
 
-Apollo Server has a peer dependency on [`graphql`](https://www.npmjs.com/package/graphql) (the core JS GraphQL implementation), which means that you are responsible for choosing the version installed in your app. In Apollo Server 3, the minimum supported version of `graphql` is v15.3.0. (Apollo Server 2 supported `graphql` v0.12 through v15.) If you are using an older version of `graphql`, you should upgrade it to the latest version before upgrading to Apollo Server 3.
+Apollo Server has a peer dependency on [`graphql`](https://www.npmjs.com/package/graphql) (the core JS GraphQL implementation), which means you are responsible for choosing the version installed in your app.
+
+**Apollo Server 3 supports `graphql` v15.3.0 and later.** (Apollo Server 2 supported `graphql` v0.12 through v15.)
+
+If you're using an older version of `graphql`, you should upgrade it to the latest version before upgrading to Apollo Server 3.
 
 ## Removed integrations
 
-Apollo Server 2 had built-in enabled-by-default integrations with the `subscriptions-transport-ws` and `graphql-upload` packages. These integrations are not part of Apollo Server 3, but you can integrate them directly in your app instead.
+Apollo Server 2 provides built-in support for subscriptions and file uploads via the `subscriptions-transport-ws` and `graphql-upload` packages, respectively.
+
+Apollo Server 3 removes these built-in integrations, in favor of enabling users to provide their own mechanisms for these features. You can still enable these integrations as they exist in Apollo Server 2.
+
 ### Subscriptions
 
-Apollo Server 2 had a superficial integration with the `subscriptions-transport-ws` package. This allowed you to add a WebSocket-based GraphQL subscriptions server alongside Apollo Server's HTTP-based GraphQL query and mutation server. This integration didn't work with Apollo Server's plugin system, Studio usage reporting, or federation.
+Apollo Server 2 provides limited, built-in support for WebSocket-based GraphQL subscriptions via the `subscriptions-transport-ws` package. This integration is incompatible with Apollo Server's plugin system and Apollo Studio usage reporting.
```

Comment:
Incompatible is strong. It's not like it prevents you from using the feature; it just isn't seen by it.